common/src/main/java/com/vexsoftware/votifier/support/forwarding/redis/RedisForwardingVoteSource.java

```diff
@@ -0,0 +1,53 @@
+package com.vexsoftware.votifier.support.forwarding.redis;
+
+import com.google.gson.Gson;
+import com.vexsoftware.votifier.model.Vote;
+import com.vexsoftware.votifier.support.forwarding.ForwardingVoteSource;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+
+import java.time.Duration;
+
+/**
+ * @author AkramL
+ */
+public class RedisForwardingVoteSource implements ForwardingVoteSource {
+
+    private final JedisPool pool;
+    private final Gson gson = new Gson();
+    private final String channel;
+
+    public RedisForwardingVoteSource(RedisCredentials credentials,
+                                     RedisPoolConfiguration configuration) {
+
+        this.channel = credentials.getChannel();
+
+        JedisPoolConfig jedisPoolConfig = new JedisPoolConfig();
+
+        jedisPoolConfig.setMaxTotal(configuration.getMaxTotal());
+        jedisPoolConfig.setMaxIdle(configuration.getMaxIdle());
+        jedisPoolConfig.setMinIdle(configuration.getMinIdle());
+        jedisPoolConfig.setMinEvictableIdleTime(Duration.ofMillis(configuration.getMinEvictableIdleTime()));
+        jedisPoolConfig.setTimeBetweenEvictionRuns(Duration.ofMillis(configuration.getTimeBetweenEvictionRuns()));
+        jedisPoolConfig.setBlockWhenExhausted(configuration.isBlockWhenExhausted());
+
+        this.pool = new JedisPool(jedisPoolConfig,
+                credentials.getHost(),
+                credentials.getPort(),
+                5000,
+                credentials.getPassword());
+    }
```

Comment:
Are you sure this works correctly when using Redis without any type of authentication? I had to do something like this for it to work on my end - though I might have done something else wrong:
```java
if (password == null || password.trim().isEmpty()) {
    this.pool = new JedisPool(jedisPoolConfig,
            credentials.getHost(),
            credentials.getPort(),
            5000
    );
} else {
    this.pool = new JedisPool(jedisPoolConfig,
            credentials.getHost(),
            credentials.getPort(),
            5000,
            credentials.getPassword()
    );
}
```

Reply:
This pull request was exactly 2 years and 6 days ago I almost forgot about it lol.
Well, never use Redis (or any database engine) without any security, but if you are running it locally then your solution should work.

Reply:
Yeah, I was just testing with a basic Redis container locally (`docker run --name redis -p 6379:6379 redis`)